site/class-1-bar/data.tsv

```diff
@@ -0,0 +1,27 @@
+letter	frequency
```

Comment:
You’re uploading files to `site/class-1-bar/`, but they should go in `site/class-1-bar/senmetsu`.